test-apps/display-test-app/package.json

```diff
@@ -13,18 +13,17 @@
   "private": true,
   "scripts": {
     "build": "npm run -s build:backend && npm run -s build:frontend",
-    "build:ci": "npm run -s build",
+    "build:ci": "npm run -s build:backend && npm run -s build:frontend-webpack",
     "build:backend": "tsc -p tsconfig.backend.json",
     "build:backend-webpack": "backend-webpack-tools build -s ./lib/backend/DtaElectronMain.js -o ./lib/build/",
-    "build:frontend": "cross-env DISABLE_NEW_JSX_TRANSFORM=true SKIP_PREFLIGHT_CHECK=true DISABLE_NEW_ASSET_COPY=true DISABLE_ESLINT_PLUGIN=true TRANSPILE_DEPS=false USE_FAST_SASS=true DISABLE_TERSER=true NODE_OPTIONS=--max_old_space_size=4096 react-scripts build",
+    "build:frontend": "tsc",
```

Comment:
any reason to actually separate these? they were previously both doing a webpack for the frontend... I'll immediately resolve this as per my other comment but we should re-visit this.